docs/self-managed/components/management-identity/access-management/access-management-overview.md

```diff
@@ -2,20 +2,26 @@
 id: access-management-overview
 title: "Manage access and permissions"
 sidebar_label: "Access and permissions"
-description: "Access Management and Permissions overview"
+description: "Learn how to manage access and permissions for Web Modeler, Console, and Optimize using Management Identity."
 ---
 
-Manage and control access to Orchestration Cluster REST APIs and custom applications using permissions and roles.
+With Management Identity, you can manage and control access to management component REST APIs and custom applications using permissions and roles.
+
+This section describes how to manage access to Web Modeler, Console, and Optimize. For access control to Orchestration Cluster components and their resources, refer to the [Orchestration Cluster authorizations](/components/concepts/access-control/authorizations.md) instead.
 
 ## About permissions
 
 When using and managing permissions, it is important to understand the following key concepts:
 
-- APIs represent the different Camunda 8 components, such as Operate, Tasklist, and so on.
+- APIs represent the different Camunda 8 management components, such as Web Modeler, Optimize, and so on.
 - Each [API defines its own set of permissions](#permissions) that to control API access.
 - Permissions are [organized using roles](./manage-permissions.md#manage-permissions-for-roles) that can be assigned to users either directly or via Groups.
 - You can also [assign permissions to your custom application](./manage-permissions.md#manage-application-permissions), such as a job worker for example.
 
+:::tip
+For detailed instructions, see the [guide about managing permissions](./manage-permissions.md).
+:::
+
 :::note
 In the Orchestration Cluster you can use [authorizations](/components/concepts/access-control/authorizations.md) to grant more fine-grained access control to Orchestration Cluster resources.
```

Comment:
This is somehow duplicated now since we have this in the intro of the page already?